xdsl/dialects/arith.py

```diff
@@ -718,13 +718,15 @@ class Cmpf(ComparisonOperation):
     predicate = prop_def(AnyIntegerAttr)
     lhs = operand_def(floatingPointLike)
     rhs = operand_def(floatingPointLike)
+    fastmath = opt_prop_def(FastMathFlagsAttr)
```

Comment:
What goes wrong with `prop_def(FastMathFlagsAttr, default = FastMathFlagsAttr("none"))`

Reply:
Apparently `default` isn't really implemented so maybe ignore this suggestion